### PR TITLE
Added test for checking event details

### PR DESCRIPTION
**Pull request by vtelt**
1.Login
2.Click on "Find event" button
3.Click on the event on "EventsExpress" page 
4.Verify the event details

### CODE REVIEW

File: src/test/java/net/eventsexpress/app/pages/EventsPage.java [hunk 1]
```diff
@@ -79,9 +83,21 @@ public EventsPage navigation(String pageName) {
         return this;
     }
 
-    public EventsPage assertUpcomingEvents() {
-        WebElement eventLogo = wait.until(ExpectedConditions.visibilityOf(upcomingEventLogo));
-        assert eventLogo.isDisplayed() : "Success message is not displayed";
+    public EventsPage assertEventsPageOpened() {
+        assert DriverManager.getCurrentUrl().contains("https://eventsexpress-test.azurewebsites.net/home/events");
```
Comment (marcinkunert) [inline src/test/java/net/eventsexpress/app/pages/EventsPage.java]: please use AssertJ for assertions

File: src/test/java/net/eventsexpress/app/pages/EventsPage.java [hunk 2]
```diff
@@ -79,9 +83,21 @@ public EventsPage navigation(String pageName) {
...
+        return this;
+    }
+
+    public EventsPage openEventDetails() {
+        WebElement eventDetailsLogo = wait.until(ExpectedConditions.visibilityOf(upcomingEventLogo));
+        assert upcomingEventLogo.isDisplayed() : "Success message is not displayed";
```
Comment (marcinkunert) [inline src/test/java/net/eventsexpress/app/pages/EventsPage.java]: please use AssertJ for assertions

File: src/test/java/net/eventsexpress/app/pages/EventsPage.java [hunk 3]
```diff
@@ -79,9 +83,21 @@ public EventsPage navigation(String pageName) {
...
+        upcomingEventLogo.click();
+        return this;
+    }
+
+    public EventsPage assertUpcomingEventDetails() {
+        WebElement eventDetailsLogo = wait.until(ExpectedConditions.visibilityOf(eventDetailsFullLogo));
+        assert eventDetailsLogo.isDisplayed() : "Success message is not displayed";
```
Comment (marcinkunert) [inline src/test/java/net/eventsexpress/app/pages/EventsPage.java]: please use AssertJ for assertions

File: src/test/java/net/eventsexpress/app/tests/EventTest.java [hunk 4]
```diff
@@ -9,13 +9,25 @@
...
         new LandingPage()
-                .findEvent();
+                .clickOnFindEventButton();
         new EventsPage()
-                .assertUpcomingEvents();
+                .assertEventsPageOpened();
+    }
+
+    @Test(priority = 2)
+    public void verifyUpcomingEventDetails() {
```
Comment (marcinkunert) [inline src/test/java/net/eventsexpress/app/tests/EventTest.java]: please add `@Story` and `@Description` allure annotations

**Review by marcinkunert — CHANGES_REQUESTED** [review]

File: src/test/java/net/eventsexpress/app/pages/EventsPage.java (hunk 1 above)
Comment (vtelt) [reply src/test/java/net/eventsexpress/app/pages/EventsPage.java]: Fixed

File: src/test/java/net/eventsexpress/app/pages/EventsPage.java (hunk 2 above)
Comment (vtelt) [reply src/test/java/net/eventsexpress/app/pages/EventsPage.java]: Fixed

File: src/test/java/net/eventsexpress/app/pages/EventsPage.java (hunk 3 above)
Comment (vtelt) [reply src/test/java/net/eventsexpress/app/pages/EventsPage.java]: Fixed

File: src/test/java/net/eventsexpress/app/tests/EventTest.java (hunk 4 above)
Comment (vtelt) [reply src/test/java/net/eventsexpress/app/tests/EventTest.java]: Fixed

**Comment by marcinkunert** [comment]
@vtelt please resolve conflicts

**Comment by vtelt** [comment]
fixed conflicts

**Review by marcinkunert — APPROVED** [review]

**Pull request merged**